Formula/rtl_433.rb

```diff
@@ -0,0 +1,41 @@
+class Rtl433 < Formula
+  desc "Program to decode radio transmissions from devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433.git",
+    tag:      "20.11",
+    revision: "0bb4672b1afb716d107b3060afc2d80a2159dbef"
+  license "GPL-2.0-or-later"
+  head "https://github.com/merbanan/rtl_433.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "librtlsdr"
+  depends_on "libusb"
+
+  resource("test_cu8") do
+    url "https://raw.githubusercontent.com/merbanan/rtl_433_tests/master/tests/oregon_scientific/uvr128/g001_433.92M_250k.cu8"
+    sha256 "7aa07b72cec9926f463410cda6056eb2411ac9e76006ba4917a0527492c5f65d"
+  end
+
+  resource("expected_json") do
+    url "https://raw.githubusercontent.com/merbanan/rtl_433_tests/master/tests/oregon_scientific/uvr128/g001_433.92M_250k.json"
+    sha256 "5054c0f322030dd1ee3ca78261b64e691da832900a2c6e4d13cc22f0fbbfbbfa"
+  end
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    resource("test_cu8").stage testpath
+    resource("expected_json").stage testpath
+
+    expected_output = shell_output("cat #{testpath}/g001_433.92M_250k.json")
```

Comment:
```suggestion
    expected_output = (testpath/"g001_433.92M_250k.json").read
```
I don't think you need `cat` here.